src/games/tictactoe/ai.ts

```diff
@@ -6,19 +6,45 @@ interface IPlayState {
   ctx: any;
 }
 
+class TictactoeRandomBot {
+  async play(state: IPlayState, playerID: string) {
+    const cell = this.generateMove(state);
+    return this.makeMove(cell);
+  }
+  generateMove(state: IPlayState) {
```

Comment:
Would be good to say it is random in the name

Reply:
I actually had that and then I decided to name the class "TictactoeRandomBot".  But I agree, I'll rename the method.